hbase-client/src/main/java/org/apache/hadoop/hbase/filter/ParseConstants.java

```diff
@@ -244,6 +244,13 @@
   public static final byte [] regexStringType = new byte [] {'r','e','g','e', 'x',
                                                              's','t','r','i','n','g'};
 
+  /**
+   * RegexStringNoCaseType byte array
+   */
+  public static final byte [] regexStringNoCaseType = new byte [] {'r','e','g','e','x',
```

Comment:
Not your fault but usually we capitalize static variable names. This is ok for now. We can change in another PR

Reply:
Yep, let's do it on a next PR. Let me merge this now,.